ssawg_trending_scraper.py

```diff
@@ -140,9 +140,17 @@ def get_proper_urls_perigee(page):
         else:
             raise RuntimeError(f'Investigate issues with {self.page}')
 
+        # Get URL text
         self.url_text = self.page_request.text
+        # Get soup from URL text
         self.soup = BeautifulSoup(self.url_text, "lxml")
-
+        # Replace relative links with absolute links; except celmon, which has no
+        # links to pull and causes an error with current implementation below
+        if self.page != "celmon":
+            for local_link in self.soup.find_all('a'):
```

Comment:
What breaks in the case of `celmon`?  I would think the `find_all` would just return an empty list if there are no matching tags.

Reply:
It's weird. I end up with a key_error, but I can't seem to find out why. 
`    return self.attrs[key]
KeyError: 'href'`
The closest thing I could attribute (and it's loose) are the Tycho2 and ICRS links in the page (https://cxc.cfa.harvard.edu/mta/ASPECT/celmon/).